mozilla/fx-private-relay
# MPP-3726 Set and get when user profile phone was registered

# New feature description
To supply helpful data to users on Relay add-on's phone tab, provide when the user set registered their phone.
`/api/v1/profiles` has a new key `date_phone_registered` that:
- If user does not have RealPhone set return None
- If user does have RealPhone set return verified_date
- If user does have RelayNumber and created_at date is not None, return created_at

# How to test
Go to `/api/v1/profiles` and check that there is `date_phone_registered` that returns the values described in above section

# Checklist (Definition of Done)
- [ ] Commits in this PR are minimal and [have descriptive commit m
